### PR TITLE
Powerfist balancing

### PR DESCRIPTION
**Pull request by Wayland-Smithy**
## About The Pull Request

TLDR It's a nerf 🦀

- Way worse power consumption: 3 max setting fists per ~10000 cell capacity, or 27 low setting fists
- Worse knockback overall, and if you fist a `MOB_SIZE_BIG` Xeno you go flying instead
- Hitting a `MOB_SIZE_BIG` also does double damage though

For the record since some marine mains seem to think a 15 tile mob throw with basically no cooldown is balanced here are the most comparable xeno abilities:
- Shrike - Unrelenting Force: 6 tile throw, 50 seconds cooldown
- Crusher - Crest Toss: 4 tile throw (5 if backwards), 18 seconds cooldown

## Why It's Good For The Game

Balances a new feature. Once toned down other aspects can be looked at for the Powerfist, maybe it could be used like spring boots to launch off walls and cades as well as xenos.

I think the need for changing justifies itself. You can still carry like a dozen of the things and also plenty of backup power.

## Changelog
:cl:
balance: Increased Powerfist power cell depletion significantly.
balance: Increased Powerfist damage vs big Xenos but reversed the knockback, and reduced the fling distance in general.
spellcheck: Fixed a Powerfist hit message typo.
/:cl:

### CODE REVIEW

**Comment by fernandos13** [comment]
so basically this makes (IF you got the HIGH CAPACITY power cell.. but wait)
1 power level = 27 punches       _[10 000/370]              = 27.027_
2 power level = 6 punches         _[10 000 / (370*2*2)]  = 6.756_
3 power level  = 3 punches        _[10 000 / (370*3^3)] = 3.003_
3 punches at power 3....
what if.. you have the .. lasgun cell. like most marines 
1000 of power
1 power level = 2 punches       _[1000/370]              = 2.702_ WAIT two punches?
2 power level = 0 punches       _[1000 / (370*2*2)]  = 0.675_ oh ... not even one..?
lets give it a extra ok.. lasgunHighcap
1600
1 power level = 2 punches       _[1600/370]             = 4.324_ four WEAK punches... thats a bit oof..?
2 power level = 1 punches       _[1600 / (370*2*2)]  = 1.081_ i mean.. at least ... ONE..?
3 power level = 0 punches       _[1600 / (370*3*3)]  = 0.480_ and again.. none

so i think you went to overboard with it.
not sure if powerfist needs a nerf.. but this is overkill for sure. even more if you count you need a screwdriver to take out batteries....

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Instead of nerfing power usage make it only take lasgun batteries
Dont make big mobs throw the user back, make it either a decreased launch range and/or more damage instead
State the new max range for punches
Also explain why you are nerfing self punches

**Comment by SilencedMP5A5** [comment]
i like the big xenos having knockback resist as funny as it is to send crushers and queens careening into the sunset
make each hit cost 1,000/2000 so you only get 10-5 hits with it with a highcap cell
keep the yeet as is for smaller xenos

and then this is a decent nerf imo

**Comment by BraveMole** [comment]
Worse knockback overall, and if you fist a MOB_SIZE_BIG Xeno you go flying instead

Reducing knockback ok for me, reversing it meh

**Comment by fernandos13** [comment]
i spend all the time i had since last post thinking around it. and i have an idea will push a PR a soon as i'm done testing thoroughly
thing is
the push power will be inverse to the DAMAGE power
it will ONLY take lasgun cells
but you will be able to **recharge if you drag to another cell "like the defib with medical backpack"**
it will be 12 punches no matter the setting
1 will be 8 tiles of distance. and 1/2 the original damage for setting one
2 will be 4 tiles of distance. and same damage as original
3 will be 2 tiles of distance. and double the original damage

how about it?

**Comment by Wayland-Smithy** [comment]
> Instead of nerfing power usage make it only take lasgun batteries
> Dont make big mobs throw the user back, make it either a decreased launch range and/or more damage instead
> State the new max range for punches
> Also explain why you are nerfing self punches

I will never support the physics defying mechanic of a puny human in armor being able to exert a force onto a big xeno thereby negating xeno intent priority for the slow ones who move less times per second than the user can click on them with melee cooldown.

**Comment by fernandos13** [comment]
also SUPPOSEDLY the powerfist does not NEED to deft physics. the REASON it actually USES energy is not NESCESSARELY to increase the forces between TWO BODIES

it can AND is most likely the case. SELF PROPELATE

a lot like a rocket. with """"theoretical ION thrusters""""

you can convert ENERGY to MOTION. and propel the FIST itself towards the target and the "gasoline" energy in the other way

no need to defy physics. you can IRL use a thruster IRL and if you use it against yourself you will .. PROPEL yourself the opposite way. not laws broken the force that makes the thruster "push" stuff is whats "pushing you"
a lot like a rocket yet. the rocket fly cause the FUEL is burned and pushes the rocket fowards. if the rocket de attaches the rocket and move it. it will be pushed in the opposite direction that the thruster is pointed. EVEN if it points at itself

**Comment by fernandos13** [comment]
also no point in having setting 4... if [10000 / (370*4*4)] = 1.689
leaving it at ONCE PUNCH per HIGH CAP powercell
that will push things AT BEST 5 tiles...

**Comment by Wayland-Smithy** [comment]
> FUEL is burned and pushes the rocket fowards.

@fernandos13 exactly. There is EXAUST being forced out the end of the rocket exerting a FORCE on the rocket. With the fist you know what is has to take that opposite reaction force? Your arm and body. There is no propulsion involved just a piston converting electrical to kinetic force between two bodies and the less massive one should lose.

**Comment by fernandos13** [comment]
thing is.. is this universe we literally got teleportation "asrs. and medvac"
wouldn't be surprising for the fist to have some propulsion instead of the mentioned piston """also if it was just a piston.. it wouldn't fair almost any better than a regular punch. and.. that would be close to the lamest weapon in game.. and hell at setting oen the fist already does 10 damage.. thats.. 5 above regular punch. you can do more with a toolbox even. the HUGE thing about power fist is the push mechanic"

**Comment by Wayland-Smithy** [comment]
I'm not advocating for staunch adherence to realism, and the two examples you cited for "fun over realism" are good ones that also have near **zero** direct effect on balance and never produce a grain of salt.

I am just arguing for some sanity. No way should this thing be allowed to throw mobs (especially large ones) 15 tiles when explosions, you know things that actually release a ton of energy near instantaneously, only have like 1/3rd the effect. Plus the negligible cooldown and comparable xeno counter abilities being so much worse only makes it seem that much more silly.

**Comment by fernandos13** [comment]
all the propell thing i was mentioning is mostly due to "self punch"

but ye 15 tiles for big benos is a but too much
but its definetly ok for smaller ones

took me 12 punches in this last game """ask the beno glitch for more info even"""

just to KEEP away the beno so i could treat a marine with hypos... 

and not that 12 punches got it crit no.. it just pushed it away... like.. 15 tiles away.... and it kept coming back.. if not for those tiles.. there would be no hope to hypo the dude ever...

i couldn't kill it. but i just wanted it to be kept far. ""setting 3 btw""
beno was.. a drone i think or sentinel

one of the t2 that spits stun acid

**Comment by fernandos13** [comment]
```
You are hit by the neurotoxic spit in the chest!
CPL  Case Huxley screams!


Young Sentinel (Glitch) slashes Case Huxley!


Case Huxley's powerfist shudders as they punch Young Sentinel (Glitch), flinging them away!
Case Huxley rams Young Sentinel (Glitch) with the powerfist!


PFC Hans Grosse (as Hans Grosse) stammers, "FUC-CK-K-K-K Y-Y-Y-YOU."


Young Sentinel (Glitch) slashes Case Huxley!


Something feels like it shattered in your right arm!
CPL  Case Huxley screams!


Young Sentinel (Glitch) spits at the dense ice wall!
You are shaken by the sudden impact!
You are hit by the neurotoxic spit in the left leg!


Case Huxley's powerfist shudders as they punch Young Sentinel (Glitch), flinging them away!
```

just some of the 12 punches ... and as you see it could came back faster than the dude on the ground would say a word

**Comment by Losenis** [comment]
If the issue is the knockback then tone it down rather than the power consumption. The machete already does more damage anyway, but isn't picked as often because it doesn't have the nodrop toggle that you seriously need in melee. The knockback is also purely situational, 99% of the time I see people punching xenos offscreen to safety to heal rather than maneuver around them to punch them into the murderball. The powerfist does low enough damage to make it a good emergency tool to punch marines away from a xeno murderball, given the damage.

Give the machete a lanyard dammit.

**Comment by fernandos13** [comment]
hey wey i think i got a idea you will like

have CTRL click change between damage and push mode.

in damage mode you get the changes you made in PUSH mode you get the old fist. """but with less damage.. just flat 10 or 15 damage no matter the setting"""" but still fling stuff far

**Comment by Swagile** [comment]
> realism as an argument to completely nerf a weapon so much that you might as well be commenting it out of the code

what.
